public/components/rating.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import $ from 'jquery';
+
+class Rating extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      ratings: {}
+    };
+
+    this.fetchRatings();
+  }
+
+  fetchRatings() {
+    $.ajax('/rooms/' + this.props.listing_id + '/ratings/', {
+      method: 'GET',
+      dataType: 'json',
+      success: (data) => {
+        this.setState({
+          ratings: data
+        });
+      }
+    });
+  }
+
+  render() {
+    return <h1> Rating Component: {
+      this.state.ratings.star_rating
+    } </h1>;
+  }
+}
+
+export default Rating;
```

Comment:
I believe the linter or maybe the CI considers it better to have a new line at the end of each file